### ISSUE TITLE
Statements below shortcuts are being skipped

### ISSUE THREAD

**Issue by McJones** [YarnSpinnerTool/YarnSpinner#188]
Any statements indented below a shortcut option are being skipped instead of being displayed when their option is selected.
So the following Yarn code:

```
Sally: What's happening?

-> Not much
    Player: Just chilling
    Sally: Yeah
-> Everything
    Sally: Anything I can help with?
    Player: No

Player: Bye
```
will show the `Sally: What's happening line` and then offer the options but regardless of what is selected just the `Player: bye` line is shown.

**Comment by joshhills** [comment]
Hi, an additional case where this happens - in `ship.yarn` as per the tutorial, when adding the shortcut block to the end e.g.

```
<<if $should_see_ship is true and $sally_warning is false>>
    Player: Sally said you wanted to see me? #line:ship1
    <<setsprite ShipFace happy>> 
    Ship: She totally did!! #line:ship3
    <<setsprite ShipFace neutral>> 
    Ship: She wanted me to tell you... #line:ship4
    Ship: If you ever go off-watch without resetting the console again... #line:ship5
    <<setsprite ShipFace happy>>
    Ship: She'll flay you alive! #line:ship6
    <<set $sally_warning to true>>
    <<wait 1>>
    Player: Uh. #line:ship7
    <<setsprite ShipFace neutral>>
<<endif>>

Ship: Anything else I can help with? #line:03cf651

-> No, thanks. #line:094f22c
    Ship: Aw, ok! #line:0f36901
-> I'm good. #line:0c7c7ef
    Ship: Let me know! #line:05da7c6

Ship: Bye! #line:0bcb990
```

the lines `Ship: Anything else I can help with? #line:03cf651` and `Ship: Aw, ok! #line:0f36901` are skipped (if you choose "No, thanks"). Everything prior to that point is fine.

If I make the yarnfile an odd number of lines after including a shortcut block it throws an IndexOutOfBounds, so it seems to be skipping forward by an extra line - an off-by-one somewhere?

**Comment by desplesda** [comment]
This is now fixed in 55214cf7824b4091ada82b4667f4659f9d04ece9, and has been released in v1.0.1. Thanks for the bug report! :heart:
